### PR TITLE
Add steps for using the PowerShell extension for v3 and v4

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
# PR Summary

We've gotten a couple folks struggle to discover that we no longer support v3 and v4 in the PowerShell extension. 

Half of the time, they are unable to update to 5.1 or 7... so I've also given them a workaround to use the last version that does support those older versions of PowerShell.

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://docs.microsoft.com/powershell/scripting/community/contributing/overview) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by TylerLeonhardt** [comment]
cc @SydneyhSmith @rjmholt

**Comment by SydneyhSmith** [comment]
Looks great thanks for adding this @TylerLeonhardt

File: reference/docs-conceptual/components/vscode/using-vscode.md [hunk 1]
```diff
@@ -99,6 +99,42 @@ functioning properly. If you still have issues getting started, let us know on
...
+
+Although the current PowerShell extension [stopped supporting v3 and v4](https://github.com/PowerShell/vscode-powershell/issues/1310),
+you can still use the last version of the extension that did.
+
+> [!NOTE]
+> There will be no additional fixes to this older version of the extension. It's provided "AS IS" but
+> is available for you if you are still using Windows PowerShell v3 and Windows PowerShell v4.
+
+First, open the Extension pane and search for "PowerShell". Then click the gear and select
+"Install another version...".
```
Comment (sdwheeler) [inline reference/docs-conceptual/components/vscode/using-vscode.md]:
```suggestion
First, open the Extension pane and search for `PowerShell`. Then click the gear and select
**Install another version...**.
```

File: reference/docs-conceptual/components/vscode/using-vscode.md [hunk 2]
```diff
@@ -99,6 +99,42 @@ functioning properly. If you still have issues getting started, let us know on
...
+
+Although this will work in the forseeable future,
+Visual Studio Code could implement a change that breaks this version of the extension.
+Because of this, and lack of support, we highly recommend either:
+
+- Downloading PowerShell 7 - which is a side-by-side install to Windows PowerShell and works
+the best with the PowerShell extension
+- Upgrading to Windows PowerShell 5.1
+
+The ["Editing with Visual Studio Code"](#editing-with-visual-studio-code) section in this article
```
Comment (sdwheeler) [inline reference/docs-conceptual/components/vscode/using-vscode.md]:
```suggestion
The [Editing with Visual Studio Code](#editing-with-visual-studio-code) section in this article
```

**Comment by rjmholt** [comment]
> What about v5.0? That is still out there.

I didn't think it was; I thought it was a mandatory upgrade to 5.1?

Certainly it's not tested with 5.0 and we haven't seen anyone use it.

**Comment by sdwheeler** [comment]
> > What about v5.0? That is still out there.
> 
> I didn't think it was; I thought it was a mandatory upgrade to 5.1?
> 
> Certainly it's not tested with 5.0 and we haven't seen anyone use it.

OK, I think it is still supported under the Windows Support Lifecycle. But we may have a case for requiring 5.1.

**Comment by TylerLeonhardt** [comment]
We can add stuff about 5.0 if someone asks - telemetry has shown no usage on 5.0 so we can defer.

**Pull request merged**